locales/en-US/translation.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I'm not sure if having the translations in .js files is possible (I could look some more into it), but what do you think about something like https://github.com/sindresorhus/gulp-strip-json-comments ?
Since the json-files need to be copied to the build folder anyway, they might as well have their comments stripped on the way.